### ISSUE TITLE
Warning: Prop `className` did not match. Server: "StyledRect-sc-1uso172-0 eRDgKF rect single-resizer" Client: "StyledRect-sc-1uso172-0 gblcbk rect single-resizer"

### ISSUE THREAD

**Issue by RouillerRomain** [mockingbot/react-resizable-rotatable-draggable#20]
Hello,
im using this module with Next.js and react and i get this error 
: Warning: Prop `className` did not match. Server: "StyledRect-sc-1uso172-0 eRDgKF rect single-resizer" Client: "StyledRect-sc-1uso172-0 gblcbk rect single-resizer"

 when i first load the page, also the class gblcbk is not added on the ResizableRect component,
 but wehn i make a change on my code and then save, it renders properly,
so basically for the first load i dont get the compoent with this class : gblcbk
any ideas how i can resolve this ?
thanks
